workflow/kfdrc-rnaseq-workflow.cwl

```diff
@@ -0,0 +1,339 @@
+cwlVersion: v1.0
+class: Workflow
+doc: |
+  # Kids First RNA-Seq Workflow
+
+  This is the Kids First RNA-Seq pipeline, which includes fusion and expression detection.
+
+  ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+  ## Introduction
+  This pipeline utilizes cutadapt to trim adapters from the raw reads, if necessary, and passes the reads to STAR for alignment.
+  The alignment output is used by RSEM for gene expression abundance estimation.
+  Additionally, Kallisto is used for quantification, but uses pseudoalignments to estimate the gene abundance from the raw data.
+  Fusion calling is performed using Arriba and STAR-Fusion detection tools on the STAR alignment outputs.
+  Metrics for the workflow are generated by RNA-SeQC.
+  **A new feature added as of February 27, 2020**, is an added workflow to run the artifact filtering portion of [annoFuse](https://www.biorxiv.org/content/10.1101/839738v1).
```

Comment:
perhaps remove this reference here...we should talk to Krutika and JoLynne about making annoFuse a Cavatica public app though...

Reply:
Removed the two feature update notes.